edf.md

```diff
@@ -0,0 +1,70 @@
+# Envelope data field
+
+> Version: 0.1.0 (Draft)
+> 
+> Authors: Oskar Thorén <oskar@status.im>, Dean Eigenmann <dean@status.im>
+
+## Table of Contents
```

Comment:
Can come in a follow up PR (issue) I suppose, but would be a good idea to be more clear about design goals here. E.g. see something like https://code.briarproject.org/briar/briar-spec/blob/master/protocols/BTP.md re design requirements, adversary model, underlying, pre-reqs.

Reply:
@oskarth does that really belong in here?

Reply:
What's the motivation for the spec then? If it can't promise any guarantees under conditions or state requirements, then what is it?